server/src/com/mirth/connect/server/userutil/DestinationSet.java

```diff
@@ -43,6 +47,7 @@ public DestinationSet(ImmutableConnectorMessage connectorMessage) {
                 this.metaDataIds = (Set<Integer>) connectorMessage.getSourceMap().get(Constants.DESTINATION_SET_KEY);
             }
         } catch (Exception e) {
+            metaDataIds = new HashSet<>();
```

Comment:
Is the catch block even necessary here?

Reply:
In the rare case of a ClassCastException when pulling out the SourceMap. Until they fix that, it's worth leaving.